app/routes.js

```diff
@@ -8,139 +8,108 @@ import { errorHandler } from './pages/error/errorHandler';
   const { solutionId, dashboardId } = req.params;
   logger.info(`navigating to Solution ${solutionId} dashboard: ${dashboardId}`);
-  try {
-    const context = await getSubDashboardPageContext({ solutionId, dashboardId });
-    res.render('pages/dashboard/subDashboards/template', context);
-  } catch (err) {
-    next(err);
-  }
-});
+  const context = await getSubDashboardPageContext({ solutionId, dashboardId });
+  res.render('pages/dashboard/subDashboards/template', context);
+}));
 
-router.get('/solution/:solutionId/dashboard/:dashboardId/section/:sectionId', async (req, res, next) => {
+router.get('/solution/:solutionId/dashboard/:dashboardId/section/:sectionId', withCatch(async (req, res) => {
   const { solutionId, dashboardId, sectionId } = req.params;
   logger.info(`navigating to Solution ${solutionId}: dashboard ${dashboardId}: section ${sectionId}`);
-  try {
-    const context = await getSectionPageContext({ solutionId, dashboardId, sectionId });
-    res.render('pages/section/template', context);
-  } catch (err) {
-    next(err);
-  }
-});
+  const context = await getSectionPageContext({ solutionId, dashboardId, sectionId });
+  res.render('pages/section/template', context);
+}));
 
-router.post('/solution/:solutionId/dashboard/:dashboardId/section/:sectionId', async (req, res, next) => {
+router.post('/solution/:solutionId/dashboard/:dashboardId/section/:sectionId', withCatch(async (req, res) => {
   const { solutionId, dashboardId, sectionId } = req.params;
   const sectionData = req.body;
-  try {
-    const response = await postSection({
-      solutionId, sectionId, sectionData, dashboardId,
-    });
-    if (response.success) {
-      res.redirect(response.redirectUrl);
-    } else {
-      const context = await getSectionPageErrorContext({
-        solutionId, sectionId, sectionData, validationErrors: response, dashboardId,
-      });
-      res.render('pages/section/template', context);
-    }
-  } catch (err) {
-    next(err);
+  const response = await postSection({
+    solutionId, sectionId, sectionData, dashboardId,
+  });
+  if (response.success) {
+    return res.redirect(response.redirectUrl);
   }
-});
+  const context = await getSectionPageErrorContext({
+    solutionId, sectionId, sectionData, validationErrors: response, dashboardId,
+  });
+  return res.render('pages/section/template', context);
+}));
 
-router.get('/solution/:solutionId/preview', async (req, res, next) => {
+router.get('/solution/:solutionId/preview', withCatch(async (req, res) => {
   const { solutionId } = req.params;
   logger.info(`navigating to Solution ${solutionId} preview`);
-  try {
-    const context = await getPreviewPageContext({ solutionId });
-    res.render('pages/preview/template', context);
-  } catch (err) {
-    next(err);
-  }
-});
+  const context = await getPreviewPageContext({ solutionId });
+  res.render('pages/preview/template', context);
```

Comment:
These are looking so much cleaner now without having to do that try catch all the time 👍